README.md

```diff
@@ -1,70 +1,55 @@
-# Getting Started with Create React App
+# Book Store
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Bookstore project, users can create books where they give the book a title and a category, and they can remove  the books.
+## Screenshots of the app.
 
-## Available Scripts
+![image](./src/img/screen.png)
 
-In the project directory, you can run:
 
-### `npm start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+[Live Demo Link](https://book-storee.herokuapp.com/)
```

Comment:
Seems you need to update your live demo link.
Push your updated code to Heroku.

Reply:
Done.